tomcat8.0/src/main/java/com/linecorp/armeria/server/http/tomcat/ArmeriaWebResourceRoot.java

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
Copied from tomcat module. I'm thinking how do I share the code between tomcat module and tomcatXX module.